config/environments/test.rb

```diff
@@ -25,33 +25,37 @@
   config.eager_load = ENV['CI'].present?
 
   # Configure public file server for tests with Cache-Control for performance.
-  config.public_file_server.enabled = true
```

Comment:
I think this is the default, and this explicit config is not in the default Rails config, so let's remove it.